SECURITY.md

```diff
@@ -2,33 +2,40 @@
 
 ## Security
 
-Microsoft takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [Microsoft](https://github.com/Microsoft), [Azure](https://github.com/Azure), [DotNet](https://github.com/dotnet), [AspNet](https://github.com/aspnet), [Xamarin](https://github.com/xamarin), and [our GitHub organizations](https://opensource.microsoft.com/).
+Microsoft takes the security of our software products and services seriously, including all source code repositories managed through our GitHub organizations, such as:
 
-If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
+* [Microsoft](https://github.com/Microsoft)
+* [Azure](https://github.com/Azure)
+* [DotNet](https://github.com/dotnet)
+* [AspNet](https://github.com/aspnet)
+* [Xamarin](https://github.com/xamarin)
+* [our GitHub organizations](https://opensource.microsoft.com/)
+
+If you believe you've found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described in the following section.
 
 ## Reporting Security Issues
 
-**Please do not report security vulnerabilities through public GitHub issues.**
+***Please do not report security vulnerabilities*** through public sites such as GitHub issues.
```

Comment:
Can I append
Making a security vulnerability public makes it easy for hackers to discover the exploit.

Reply:
I think this text was all added by default by the standard MS OSS bot when the repo was created. Should that bot/process be updated with any changes we're making here?